From 2a784451090bd6e8681b397270d3f99c117d2a1e Mon Sep 17 00:00:00 2001
From: Sebastian Muxel <sebastian@muxel.dev>
Date: Sat, 11 Oct 2025 22:54:50 +0200
Subject: [PATCH] dts: rockchip: rk3576-luckfox-core3576: set output to UART2

This is because UART0 is put on the most stupid header ever and i can't
access it right now.

Upstream-Status: Inappropriate [oe specific]
Signed-off-by: Sebastian Muxel <sebastian@muxel.dev>
---
 arch/arm64/boot/dts/rockchip/rk3576-luckfox-core3576.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3576-luckfox-core3576.dtsi b/arch/arm64/boot/dts/rockchip/rk3576-luckfox-core3576.dtsi
index 9187012d6fa4..04c09fcc2c26 100644
--- a/arch/arm64/boot/dts/rockchip/rk3576-luckfox-core3576.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3576-luckfox-core3576.dtsi
@@ -21,7 +21,7 @@ aliases {
 	};
 
 	chosen {
-		stdout-path = "serial0:1500000n8";
+		stdout-path = "serial2:1500000n8";
 	};
 
 	hdmi-con {
